### PR TITLE
fix(wallet-mobile): collateral dialog closing

### PR DESCRIPTION
**Pull request by banklesss**

### CODE REVIEW

File: apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx [hunk 1]
```diff
@@ -95,6 +95,8 @@ export const ManageCollateralScreen = () => {
   const isLoading = isLoadingTx || isLoadingCollateral
 
   const handleGenerateCollateral = () => {
+    closeModal()
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx:98]: We may need to fix also the issue where it generates new collateral even if we already generated collateral before.

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx:98]: Maybe if the collateral already exists not show the modal?

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx:98]: The button that triggers the modal is hidden when `hasCollateral` is true, do you think any further improvement is needed?

Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/useCases/changeWalletSettings/ManageCollateral/ManageCollateralScreen.tsx:98]: ok its approved thanks !

**Pull request merged**